Sources_v3/Workers/Background/MessageEditor.swift

```diff
@@ -21,20 +21,17 @@ import Foundation
 class MessageEditor<ExtraData: ExtraDataTypes>: Worker {
```

Comment:
In git-blame I see the fix for the crash in `MessageEditor` on iOS 12. 

However, the bug was solved partially: the `processNextMessage()` was scheduled on bg-context queue only from `handleChanges()` method but not from `removeMessageIDAndContinue()`. 

While resolving this issue I came up with the refactoring you see here. With such bug-fix + refactoring in place it was super simple to implement the **[weak self]** stuff